instrumentation/mybatis/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/mybatis/MybatisAttributesExtractor.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.mybatis;
+
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import javax.annotation.Nullable;
+
+class MybatisAttributesExtractor implements AttributesExtractor<MapperMethodRequest, Void> {
+
+  @Override
+  public void onStart(
+      AttributesBuilder attributes,
+      Context parentContext,
+      MapperMethodRequest mapperMethodRequest) {
+    attributes.put("component.name", "mybatis");
```

Comment:
this isn't really useful `InstrumentationScopeInfo` already contains the name of the instrumentation library

Reply:
> this isn't really useful `InstrumentationScopeInfo` already contains the name of the instrumentation library

I deleted this class.